bootstrap/eks/controllers/eksconfig_controller.go

```diff
@@ -162,3 +177,44 @@ func (r *EKSConfigReconciler) SetupWithManager(mgr ctrl.Manager, option controll
 		WithOptions(option).
 		Complete(r)
 }
+
+// storeBootstrapData creates a new secret with the data passed in as input,
+// sets the reference in the configuration status and ready to true.
+func (r *EKSConfigReconciler) storeBootstrapData(ctx context.Context, scope *EKSConfigScope, data []byte) error {
```

Comment:
Copy from https://github.com/kubernetes-sigs/cluster-api/blob/d20d38c629fd3676f9e19fe0021ec19abff9566e/bootstrap/kubeadm/controllers/kubeadmconfig_controller.go#L802 can we re-use instead?

Reply:
the kind "EKSConfig" bit may cause some problems. I think we duplicate for now and open an issue upstream to see if it makes sense to add a reusable utility func?